simcon_project/simcon_project/settings.py

```diff
@@ -140,6 +139,8 @@
 STATIC_URL = '/static/'
 
 LOGIN_REDIRECT_URL = '/redirect-from-login'
+LOGOUT_REDIRECT_URL = 'Login'
+LOGIN_URL = 'Login'
```

Comment:
should this not be` '/Login'`

Reply:
Here I'm using the name of the url. We can swap out what the address of the 'Login' url is, without having to poke everywhere it is referenced.